cmd/app.go

```diff
@@ -46,6 +46,11 @@ func Main() {
 		klog.Infof("FLAG: --%s=%q", flag.Name, flag.Value)
 	})
 
+	// Process on macOS must run using sudo
+	if runtime.GOOS == "darwin" && syscall.Geteuid() != 0 {
+		klog.Warning("Please run this again with `sudo`.")
```

Comment:
```suggestion
		klog.Fatalf("Please run this again with `sudo`.")
```

my comment about warning was with the previous approach, in case you could not get the user , apply this and we are good to go